third-party/gasnet/Makefile

```diff
@@ -51,15 +51,16 @@ CHPL_GASNET_CFG_OPTIONS += --disable-gemini --disable-aries
 endif
 endif
 ifeq ($(CHPL_MAKE_TARGET_PLATFORM),cray-xc)
-CHPL_GASNET_CFG_SCRIPT=cross-configure-crayxc-linux
+CHPL_GASNET_CFG_SCRIPT=cross-configure-cray-aries-alps
```

Comment:
There are *-slurm variants as well, but the only diference between them is the MPIRUN_CMD, which I don't think we care about (and I really don't want to have to rebuild gasnet based on the launcher, since that'll add ~20 minutes to our module build time):

`diff cross-configure-cray-aries-alps cross-configure-cray-aries-slurm`
```diff 
< MPIRUN_CMD="${APRUN:-aprun} -n %N %C" ; export MPIRUN_CMD  # launch command for MPI jobs
---
> MPIRUN_CMD="${SRUN:-srun} -K0 -W60 %V -mblock -n %N %C" ; export MPIRUN_CMD  # launch command for MPI jobs
```